tar/siginfo.c

```diff
@@ -55,7 +55,9 @@ struct siginfo_data {
 #ifdef SIGINFO
 	struct sigaction siginfo_old;
 #endif
+#ifdef SIGUSR1
```

Comment:
Technically not necessary, since POSIX says that SIGUSR1 exists.  (It is silent about SIGINFO.)

I agree with having this here for the sake of consistency, but the commit message should mention that we're doing this for consistency, not because it's actually needed.

Reply:
done